aws-blog-mirth-healthcare-hub/hl7-webapp/service-xml-to-json.js

```diff
@@ -0,0 +1,49 @@
+
+//web service to generate JSON from XML
+var http = require('http');
+var qs = require('querystring');
+var fs = require('fs');
+var xml2js = require('xml2js');
+const util = require('util');
+
+//Port to listen for requests as TCP
+const PORT=8001;
+var conn_count = 0;
+var parser = new xml2js.Parser({explicitArray : false});
+
+//We need a function which handles requests and send response
+function handleRequest(request, response){
```

Comment:
Please format this file according to java script code style.

http://standardjs.com/